website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2024
 
+- **New:** GA of [job warnings](/docs/deploy/job-notifications) is now available. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were cancelled. Now, you can also receive alerts when jobs have encountered warnings during their run. This provides more flexibility on when to be notified. 
```

Comment:
Love this! Can we change it to something like "when jobs have warnings from tests or source freshness checks"